sdk/loadtestservice/azure-developer-loadtesting/src/test/java/com/azure/developer/loadtesting/LoadTestAdministrationTests.java

```diff
@@ -0,0 +1,236 @@
+
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.MethodOrderer;
+import org.junit.jupiter.api.Order;
+import org.junit.jupiter.api.Test;
+import org.junit.jupiter.api.TestMethodOrder;
+
+@TestMethodOrder(MethodOrderer.OrderAnnotation.class)
+public final class LoadTestAdministrationTests extends LoadTestingClientTestBase {
+
+    // Helpers
+
+    private Map<String, Object> getTestBodyFromDict() {
+        Map<String, Object> testMap = new HashMap<String, Object>();
+        testMap.put("displayName", "Java SDK Sample Test");
+        testMap.put("description", "Sample Test");
+
+        Map<String, Object> loadTestConfigMap = new HashMap<String, Object>();
+        loadTestConfigMap.put("engineInstances", 1);
+        testMap.put("loadTestConfiguration", loadTestConfigMap);
+
+        Map<String, Object> envVarMap = new HashMap<String, Object>();
+        envVarMap.put("threads_per_engine", 1);
+        envVarMap.put("ramp_up_time", 0);
+        envVarMap.put("duration_in_sec", 10);
+        envVarMap.put("domain", "azure.microsoft.com");
+        envVarMap.put("protocol", "https");
+        testMap.put("environmentVariables", envVarMap);
+
+        return testMap;
+    }
+
+    private BinaryData getFileBodyFromResource(String fileName) {
+        URL url = LoadTestAdministrationTests.class.getClassLoader().getResource(fileName);
+        return BinaryData.fromFile(new File(url.getPath()).toPath());
+    }
+
+    // Puts and Patches
+
+    @Test
+    @Order(1)
+    public void createOrUpdateLoadTest() {
+        BinaryData body = BinaryData.fromObject(getTestBodyFromDict());
+        Response<BinaryData> response = builder.buildLoadTestAdministrationClient().createOrUpdateTestWithResponse(newTestId, body, null);
+        Assertions.assertTrue(Arrays.asList(200, 201).contains(response.getStatusCode()));
+    }
+
+    @Test
+    @Order(2)
+    public void uploadTestFile() {
+        BinaryData file = getFileBodyFromResource(uploadCsvFileName);
+        RequestOptions requestOptions = new RequestOptions().addQueryParam("fileType", "ADDITIONAL_ARTIFACTS");
+        Response<BinaryData> response = builder.buildLoadTestAdministrationClient().uploadTestFileWithResponse(
+                                                newTestId,
+                                                uploadCsvFileName,
+                                                file,
+                                                requestOptions);
+        Assertions.assertEquals(201, response.getStatusCode());
+    }
+
+    @Test
+    @Order(3)
+    public void beginUploadAndValidate() {
+        BinaryData file = getFileBodyFromResource(uploadJmxFileName);
+        RequestOptions fileUploadRequestOptions = new RequestOptions().addQueryParam("fileType", "JMX_FILE");
+        SyncPoller<ValidationStatus, BinaryData> poller = builder.buildLoadTestAdministrationClient().beginUploadAndValidate(newTestId, uploadJmxFileName, file, 1, fileUploadRequestOptions);
```

Comment:
Use `TestBase.setPlaybackSyncPollerPollInterval` on the poller in test (it changes the interval for playback).